### PR TITLE
Viz Dashboard Update

### PR DESCRIPTION
**Pull request by cbohn4**
Current version of the dashboard since sandhills retirement.

### CODE REVIEW

File: dashboards/hcc.erb [hunk 1]
```diff
@@ -19,47 +19,35 @@ $(function() {
...
       <div data-id="JobsCompleted" data-view="NonShortenedNumber" data-title="Jobs Completed Today"></div>
     </li>
-<!--
-    <li data-row="3" data-col="2" data-sizex="1" data-sizey="1">
-      <div data-id="HoursWeek" data-view="Number" data-title="CPU Hours This Week"></div>
-    </li>
--->
     <li data-row="3" data-col="3" data-sizex="2" data-sizey="1">
-      <div data-id="NetworkGraph" data-view="NetworkGraph" data-title="Internet2 Bandwidth" data-suffix="ps"></div>
+    <div data-id="ShorNetworkGraph" data-view="NetworkGraph" data-title="Shor Internet Bandwidth" data-suffix="ps" style="background-color:#F58A1F;"></div>
```
Comment (djw8605) [inline dashboards/hcc.erb]:
```suggestion
    <div data-id="ShorNetworkGraph" data-view="NetworkGraph" data-title="Shorr Internet Bandwidth" data-suffix="ps" style="background-color:#F58A1F;"></div>
```

Schorr has 2 r's

File: topUsersData.sh [hunk 2]
```diff
@@ -0,0 +1,5 @@
+ssh -i /home/dashboard/VizDashKey root@crane-head > squeueDataRaw.csv 
+ssh -i /home/dashboard/VizDashKey root@tusker-head >> squeueDataRaw.csv
+
+python slurmToRuby.py > squeueData.csv
+rm squeueDataRaw.csv
```
Comment (djw8605) [inline topUsersData.sh:5]: Man this is ugly.  Really, you have root ssh keys on viz to log into the head nodes?

File: slurmToRuby.py [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+numCore = 0
+userCores = {}
+with open(fName) as f:
+     lines = f.readlines()
+for i in lines:
+
+    numCore = (int(re.sub(' +', ' ', i).split(' ')[1]))
+    userName = re.sub(' +', ' ', i).split(' ')[0]
+    if userName in userCores:    
+        userCores[userName] = numCore + userCores.get(userName)
```
Comment (djw8605) [inline slurmToRuby.py:12]: I think you can just do
```suggestion
        userCores[userName] += numCore
```

File: jobs/updatenetwork.rb [hunk 4]
```diff
@@ -1,134 +1,10 @@
...
-  
-  #network_debug.close
-
-  send_event('NetworkGraph', { points: points })
-  send_event('HCCAmazonPrice', { network_bandwidth: last_point })
+(1..30).each do |i|
+    send_event('ShorNetworkGraph', { points: points}  )
+    send_event('PkiNetworkGraph', { points: points}  )
+    send_event('HCCAmazonPrice', { network_bandwidth: 1000000000000000000000000000000 })
+    sleep(0.2)
```
Comment (djw8605) [inline jobs/updatenetwork.rb:9]: Wait, so there is no actual network data?  It's all faked?

**Review by djw8605 — CHANGES_REQUESTED** [review]
Some comments.

File: dashboards/hcc.erb (hunk 1 above)
Comment (CarrieBrown) [reply dashboards/hcc.erb]: Fixed

File: jobs/updatenetwork.rb (hunk 4 above)
Comment (CarrieBrown) [reply jobs/updatenetwork.rb:9]: This is initial dummy data that was put in to try to fix the graph generating error. This is no longer an issue with the rework.

**Pull request merged**